source/boards/overview.rst

```diff
@@ -13,4 +13,30 @@ With Boards you can:
 * **Stay on schedule:** Clearly-defined tasks, owners, checklists, and deadlines
 * **Increase transparency and keep everybody in the loop:** Everything your team needs in one place, including documents, images, and links that are visible to every stakeholder
 
-Boards is also available as a separate single-user Personal Edition called Focalboard.
+What's a board?
+---------------
+
+A board is a collection of cards to help you manage your projects, organize tasks, and collaborate with your team all in one place.
+
+Boards can be displayed and filtered in different views such as kanban, table, calendar, and gallery views to help you visualize work items in the format that makes most sense to you.```
+
+What's a card?
+--------------
+
+Cards are used on a board to track individual work items. Cards are customizable and can have a number of properties added to them, which are then used as a way to tag, sort, and filter the cards.
+
+When working with cards, you can manage properties, add descriptions, attach images, assign them to team members, mention team members, add comments, and so on.
+
+Access your boards
+------------------
+
+Open the Boards tab via the product menu in the top left corner of Mattermost, to view all the boards for your team. You can select the Boards icon in the Apps Bar to open the right-hand panel, and display boards linked to the channel or message that you're in.
+
+If you don't see the Apps Bar and your boards layout looks different to what's described, you may be using an older version of Mattermost. The navigation content in this documentation will not be aligned with your navigation, but the functionality described will be applicable across versions unless otherwise noted.
+
+Focalboard personal server and desktop
```

Comment:
@wuwinson, I've added this based on some of the questions we've received about features that don't apply across the options. Can you think of any others to add?